scripts/module_list/requirements.txt

```diff
@@ -0,0 +1,5 @@
+flake8
+pytest
+mdutils
+numpy
+setuptools
```

Comment:
Why is `setuptools` an explicit dependency?

I guess only `mdutils` and `numpy` really are, and maybe there should be a separate `requirements_tests.txt` for `flake8` and `pytest`?